5-infrastructure/modules/base-env/outputs.tf

```diff
@@ -0,0 +1,501 @@
+  }
+}
+
+/******************************************
+  KMS Outputs
+*****************************************/
+
+output "kms_gke_1_keyring" {
+  description = "Self link of the keyring."
+  value       = module.kms_gke_1.keyring
+}
+
+output "kms_gke_1_keyring_resource" {
+  description = "Keyring resource."
+  value       = module.kms_gke_1.keyring_resource
+}
+
+output "kms_gke_1_keys" {
+  description = "Map of key name => key self link."
+  value       = module.kms_gke_1.keys
+}
+
+output "kms_gke_1_keyring_name" {
+  description = "Name of the keyring."
+  value       = module.kms_gke_1.keyring_name
+}
+
+output "kms_gke_2_keyring" {
+  description = "Self link of the keyring."
+  value       = module.kms_gke_2.keyring
+}
+
+output "kms_gke_2_keyring_resource" {
+  description = "Keyring resource."
+  value       = module.kms_gke_2.keyring_resource
+}
+
+output "kms_gke_2_keys" {
+  description = "Map of key name => key self link."
+  value       = module.kms_gke_2.keys
+}
+
+output "kms_gke_2_keyring_name" {
+  description = "Name of the keyring."
+  value       = module.kms_gke_2.keyring_name
+}
+
+output "kms_sql_1_keyring" {
+  description = "Self link of the keyring."
+  value       = module.kms_sql_1.keyring
+}
+
+output "kms_sql_1_keyring_resource" {
+  description = "Keyring resource."
+  value       = module.kms_sql_1.keyring_resource
+}
+
+output "kms_sql_1_keys" {
+  description = "Map of key name => key self link."
+  value       = module.kms_sql_1.keys
+}
+
+output "kms_sql_1_keyring_name" {
+  description = "Name of the keyring."
+  value       = module.kms_sql_1.keyring_name
+}
+
+output "kms_sql_2_keyring" {
+  description = "Self link of the keyring."
+  value       = module.kms_sql_2.keyring
+}
+
+output "kms_sql_2_keyring_resource" {
+  description = "Keyring resource."
+  value       = module.kms_sql_2.keyring_resource
+}
+
+output "kms_sql_2_keys" {
+  description = "Map of key name => key self link."
+  value       = module.kms_sql_2.keys
+}
+
+output "kms_sql_2_keyring_name" {
+  description = "Name of the keyring."
+  value       = module.kms_sql_2.keyring_name
+}
+
+/******************************************
+  Bastion Host Outputs
+*****************************************/
+
+output "bastion_hostname" {
+  description = "Host name of the bastion"
+  value       = module.bastion.hostname
+}
+
+output "bastion_ip_address" {
+  description = "Internal IP address of the bastion host"
+  value       = module.bastion.ip_address
+}
+
+output "bastion_self_link" {
+  description = "Self link of the bastion host"
+  value       = module.bastion.self_link
+}
+
+output "bastion_service_account_email" {
+  description = "Email address of the SA created for the bastion host"
+  value       = module.bastion.service_account_email
+}
+
+output "bastion_cidr_range" {
+  description = "Self link of the bastion host"
+  value       = module.bastion.cidr_range
+}
+
+output "bastion_subnet_name" {
```

Comment:
Let's only output a value if it is used in subsequent stages like ACM/ASM setup.